browser/main/Detail/MarkdownNoteDetail.js

```diff
@@ -262,6 +266,11 @@ class MarkdownNoteDetail extends React.Component {
     }
   }
 
+  handleGenerateToc () {
+    markdownToc.generate(this.refs.content.value,
+      (modifiedValue) => { this.refs.content.refs.code.setValue(modifiedValue) })
```

Comment:
Can you remove the `{` and `}` to retain the code style

Reply:
👍